example/kustomization.yaml

```diff
@@ -39,8 +39,8 @@ resources:
 - ../common/istio-1-16/istio-crds/base
 - ../common/istio-1-16/istio-namespace/base
 - ../common/istio-1-16/istio-install/base
-# OIDC Authservice
-- ../common/oidc-authservice/base
+# oauth
```

Comment:
as mentioned in a comment above, let's keep authservice as the default just for this release